internal/controller/common/backup_config_util.go

```diff
@@ -16,3 +34,119 @@ func GetConfigSection(config map[string]interface{}, section string) (map[string
 
 	return sectionMap, nil
 }
+
+func GetBackupServicePodList(k8sClient client.Client, name, namespace string) (*corev1.PodList, error) {
+	var podList corev1.PodList
+
+	labelSelector := labels.SelectorFromSet(utils.LabelsForAerospikeBackupService(name))
+	listOps := &client.ListOptions{
+		Namespace: namespace, LabelSelector: labelSelector,
+	}
+
+	if err := k8sClient.List(context.TODO(), &podList, listOps); err != nil {
+		return nil, err
+	}
+
+	return &podList, nil
+}
+
+func ReloadBackupServiceConfigInPods(
+	k8sClient client.Client,
+	log logr.Logger,
```

Comment:
log is not used in this function, check if there was any plan to add some logging in this.

Reply:
It'll be used in `validateBackupSvcConfigReload` func. Already added TODO to uncomment it when backup service fixes issues at their end